HelloWorld/models.py

```diff
@@ -1,3 +1,38 @@
 from django.db import models
 
 # Create your models here.
+
+class Category(models.Model):
+    name = models.CharField(max_length=100)
+    def __str__(self):
+        return self.name
+
+
+class Task(models.Model):
+
+    class Status(models.TextChoices):
+        NEW = "New"
+        IN_PROGRESS = "In progress"
+        PENDING = "Pending"
+        BLOCKED = "Blocked"
+        DONE = "Done"
+
+    title = models.CharField(max_length=100)
+    description = models.TextField(max_length=1000)
+    category = models.ManyToManyField(Category)
+    status = models.CharField(max_length=100, choices=Status.choices, default=Status.NEW)
```

Comment:
Вы сами определяете возможные варианты статусов, но они значительно короче 100 символов
```suggestion
    status = models.CharField(max_length=20, choices=Status.choices, default=Status.NEW)
```